mim/utils/utils.py

```diff
@@ -157,19 +161,23 @@ def get_content_from_url(url: str,
 @typing.no_type_check
 def download_from_file(url: str,
                        dest_path: str,
-                       hash_prefix: Optional[str] = None) -> None:
+                       hash_prefix: Optional[str] = None,
+                       check_certificate: bool) -> None:
```

Comment:
```suggestion
                       check_certificate: bool = True) -> None:
```

Reply:
Please also update the order in the docstring

Reply:
Thanks for your contribution. This PR can be merged after resolving the final comment.

Reply:
Thank you for your review!